CHANGELOG.txt

```diff
@@ -6,15 +6,16 @@ New Features
 
 - AIP-39: Add (customizable) Timetable class to Airflow for richer scheduling behaviour (#15397, #16030,
   #16352, #17030, #17122, #17414, #17552, #17755, #17989, #18084, #18088, #18244, #18266, #18420, #18434,
-  #18421, #18475, #18499, #18573)
+  #18421, #18475, #18499, #18573, #18522, #18729)
```

Comment:
```suggestion
  #18421, #18475, #18499, #18573, #18522, #18729, #18706)
```

Reply:
Gotcha, didn't realize it was related, multiple-layers away now.